torchx/Makefile

```diff
@@ -14,25 +13,28 @@ TORCHX_SO = $(PRIV_DIR)/torchx.so
 ifeq ($(shell uname -s), Darwin)
 	LDFLAGS += -flat_namespace -undefined suppress
 	POST_INSTALL = install_name_tool -add_rpath @loader_path/lib $(TORCHX_SO)
+	LIBTORCH_DOWNLOAD_URL ?= https://download.pytorch.org/libtorch/cpu/libtorch-macos-1.9.1.zip
 else  # Linux
-    CFLAGS += -fopenmp
-	LDFLAGS += -Wl,-rpath,'$$ORIGIN/lib'
+    CFLAGS += -isystem $(LIBTORCH_INCLUDE_PATH) -isystem $(LIBTORCH_API_INCLUDE_PATH) -isystem $(ERTS_INCLUDE_DIR)
+	LDFLAGS += -Wl,-rpath,$(LIBTORCH_LINK_PATH) $(LIBTORCH_LINK_PATH)/libtorch.so $(LIBTORCH_LINK_PATH)/libc10.so $(LIBTORCH_LINK_PATH)/libkineto.a -Wl,--no-as-needed,"$(LIBTORCH_LINK_PATH)/libtorch_cpu.so" -Wl,--as-needed $(LIBTORCH_LINK_PATH)/libc10.so -lpthread -Wl,--no-as-needed,"$(LIBTORCH_LINK_PATH)/libtorch.so" -Wl,--as-needed
 	POST_INSTALL = $(NOOP)
+	LIBTORCH_DOWNLOAD_URL ?= https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.9.1%2Bcpu.zip
 endif
 
 .DEFAULT_GLOBAL := build
 
-build: check_libtorch $(TORCHX_SO)
+build: libtorch $(TORCHX_SO)
 
-check_libtorch:
-	@ if [ -z "$(LIBTORCH_DIR)" ]; then \
-		echo "LIBTORCH_DIR env var is not set. It should point to the directory, where LibTorch is installed."; \
-		echo "You can download LibTorch here: https://pytorch.org/get-started/locally/"; \
-		exit 1; \
+libtorch:
```

Comment:
After we get this makefile working, we can move this to download via Elixir as suggested in #494